src/common/KernelBase.hpp

```diff
@@ -29,9 +29,40 @@
 #include <string>
 #include <vector>
 #include <iostream>
+#include <map>
 #include <limits>
 #include <utility>
 
+#if defined(RAJA_PERFSUITE_USE_CALIPER)
+
+#define CALI_START \
+    if (doCaliperTiming) { \
+      std::string kstr = getName(); \
+      std::string gstr = getGroupName(kstr); \
+      std::string vstr = "RAJAPerf"; \
+      CaliMeta(); \
+      CALI_MARK_BEGIN(vstr.c_str()); \
```

Comment:
This doesn't look right to me. Is 'gstr' the "group" name? If so, then getGroupName() should take a GroupID enum value. Is 'vstr' the "variant" name? If so, then getVariantName() should take a VariantID enum value.

Am I misunderstanding this?

Reply:
We specficially moved away from `getVariantName()` to a static string, `RAJAPerf`, so different runs will not have different top-level node names. This fits the standard expected by our tools. This information should instead be collected in the metadata, as shown [here](https://github.com/LLNL/RAJAPerf/pull/331/files#diff-f1b7ede2b59bf264efbe43bc92349deb28c8f49dc87efa11b4882ab6805beb24R582).

Reply:
OK. I'm still confused about this. I can ask you about it later sometime.